docs/pages/index.rst

```diff
@@ -9,26 +9,74 @@ When rendering a scene from the Matterport3D dataset, *Habitat-Sim* achieves
+.. class:: m-table m-fullwidth
 
-Checkout our `ECCV 2020 tutorial series`_ for quick-start code and video overviews of many core features!
+=================================================== ======================================================================================== ======================
+Basics for Navigation                               `Video <https://youtu.be/kunFMRJAu2U?list=PLGywud_-HlCORC0c4uj97oppQrGiB6JNy>`__                                     `Interactive Colab <https://colab.research.google.com/github/facebookresearch/habitat-sim/blob/master/examples/tutorials/colabs/ECCV_2020_Navigation.ipynb>`__
 
-.. _ECCV 2020 tutorial series: https://aihabitat.org/tutorial/2020/
+Interaction                                         `Video <https://youtu.be/6eh0PBesIgw?list=PLGywud_-HlCORC0c4uj97oppQrGiB6JNy>`__                                     `Interactive Colab <https://colab.research.google.com/github/facebookresearch/habitat-sim/blob/master/examples/tutorials/colabs/ECCV_2020_Interactivity.ipynb>`__
 
-.. block-info:: C++ API documentation
+Advanced Topics                                     `Video <https://youtu.be/w_kDq6UOKos?list=PLGywud_-HlCORC0c4uj97oppQrGiB6JNy>`__                                     `Interactive Colab <https://colab.research.google.com/github/facebookresearch/habitat-sim/blob/master/examples/tutorials/colabs/ECCV_2020_Advanced_Features.ipynb>`__
 
-    Habitat-sim is designed to be used primarily through its Python API. As such, the
-    end-user tutorials and docs linked above focus on Python.
+Profiling and Optimization                          `Video <https://youtu.be/I4MjX598ZYs?list=PLGywud_-HlCORC0c4uj97oppQrGiB6JNy>`__                                     `Interactive Colab <https://colab.research.google.com/gist/eundersander/b62bb497519b44cf4ceb10e2079525dc/faster-rl-training-profiling-and-optimization.ipynb>`__
 
-    If you're looking for API reference of Habitat-sim's C++ internals, please see the
-    `C++ API <cpp.html>`_ tab.
+New Actions                                         :ref:`Web page <std:doc:new-actions>`
+
+Notebooks                                           :ref:`Web page <std:doc:notebooks>`
+
+Creating a stereo agent                             :ref:`Web page <std:doc:stereo-agent>`
+
+Working with light setups                           :ref:`Web page <std:doc:lighting-setups>`
+
+Extracting Images                                   :ref:`Web page <std:doc:image-extractor>`
+
+Interactive Rigid Objects                           :ref:`Web page <std:doc:rigid-object-tutorial>`
+=================================================== ======================================================================================== ======================
+
+Python Classes
+==============
+
+See the `Classes <./classes.html>`_ tab.
+
+Python Unit Tests
+=================
+
+Browse selected unit tests that demonstrate essential *Habitat-Sim* interfaces.
```

Comment:
My opinion on this section is that linking to some unit tests is a low-effort way to show new users some Habitat API usage. However, if some of these files are not self-explanatory enough, they'll just further confuse a new user. This list doesn't need to be exhaustive, and in fact I'd be happy to remove more entries here (i.e. be more selective).

Reply:
This looks good to me. 👍  We may want to clean up and comment some of these a bit in the near future.

Reply:
Tests and CI are my first to go source of how to use something or install. While that is less welcoming for users.
I support having links and mentions of tests/examples that can be useful for understanding.